android/app/src/main/java/com/couchbase/todo/TasksFragment.java

```diff
@@ -111,7 +112,9 @@ public void onAttach(@NonNull Context context) {
 
     @Override
     public void onActivityResult(int requestCode, int resultCode, Intent data) {
-        if ((resultCode != Activity.RESULT_OK) || (requestCode != REQUEST_TAKE_PHOTO)) { return; }
+        if ((resultCode != Activity.RESULT_OK) || (requestCode != REQUEST_TAKE_PHOTO)) {
+            return;
+        }
```

Comment:
I think this is an auto-format issue, let me know if i should fix this so it looks like the original format